### ISSUE TITLE
Ability to see max scores 

### ISSUE THREAD

**Issue by Dr-Kakashi** [Hanabi-Live/hanabi-live#1092]
I'm not sure if my title is correct, but I'm looking for a way to pull up the history of max scores for a variant and be able to see how other teams did for that seed.  

For example, no variant just lists the 20 or so most recent games.  

It would be nice to pull a list where a seed has been played by many teams to see how well each team did compared to one another.

**Comment by Dr-Kakashi** [comment]
![image](https://user-images.githubusercontent.com/48993588/82787380-20cf8d00-9e1b-11ea-86f0-af7d267efe0c.png)

For Example, Clue Starved (6 Suits).  It says there are 6 perfect scores.  Is there a way to pull the list to see them easily?  Right now it's just daunting to try to find it.  

I guess another way to look at it is....a leaderboard of the games where teams have successfully gotten a max score.  

https://hanabi.live/variant/51

The ability to switch between Recent Games and Maxed Score Games.

**Comment by zsole2** [comment]
Instead of this switch, this could be handled by making the variant tables sortable, which is the case with the stats page or personal profile pages.

Alternatively, a filter field, like the one on profile/missing-scores, may be added for the score.

The simplest solution would be to use ctrl-f, but that would require listing all games instead of recent games. Can be a lot of rows to load, but currently there are about 10 variants with more games than the number of variants on the stats page.

**Comment by Dr-Kakashi** [comment]
Would be nice if there was like.....a separate tab just showing highest/maximal scores achieved in the variant page and we can sort it by date completed or fewest # of turns (competition format) in order to easily compare.  

![image](https://user-images.githubusercontent.com/48993588/88996126-540e1c80-d2a1-11ea-82b7-faeb56114308.png)

Like here for example.  If you check the 3p scores, you can see the team that only 29 at the top of the list
If you check the 4p scores you can see the teams that got 30
Basically, any other score other than the maximum will not be shown

**Comment by DrAnax** [comment]
This issue should be closed after the ajax API. Variants can be filtered by score and all previous games are now shown.
